boards/nordic/nrf52840dk/test.config.toml

```diff
@@ -0,0 +1,30 @@
+# Test configuration file to modify test environment and settings
+# There will be a file like this in EACH board folder.
```

Comment:
What reads this config file? "test" is somewhat generic, is there a more unique name that would be appropriate?